backend/rest_api_service/main.py

```diff
@@ -61,4 +61,125 @@ async def get_all_participants(response: Response, db: Session = Depends(get_db)
 
     except Exception as e:
         response.status_code = 500
-        return {"message": f"An error occured: {e}"}
+        return {"message": f"An error occurred: {e}"}
+
+
+@app.get("/event/{event_id}", tags=['Event'], status_code=status.HTTP_200_OK)
+async def get_event_by_id(event_id, response: Response, db: Session = Depends(get_db)):
+    try:
+        return await crud.get_event_by_id(event_id=event_id, db=db)
```

Comment:
w ramach czystości kodu ja bym zrobił event=await... i dopiero potem return

Reply:
@GrzesS0 nadal aktualne

Reply:
Ja zdecydowanie wolę taki zapis. Jak dla mnie, to jest czytelniejszy

Reply:
ale to nawet nie chodzi o zapis, tylko jak robisz `return await` to zwracas couritine, trzeba to przypisać do zmiennej i potem dopiero zwrócić